pkg/crc/machine/driver_windows.go

```diff
@@ -37,6 +38,7 @@ func getDriverOptions(machineConfig config.MachineConfig) interface{} {
 	switch machineConfig.VMDriver {
 
 	case "virtualbox":
+		output.Outln("Virtualbox support is deprecated and will be removed in the next release.")
```

Comment:
Instead of `output.Outln` I would use `logging.Warn` so that in the output it atleast show a warning to the user about this deprecation.

Reply:
I was considering this... If you think this is better, I'll change